src/components/Datepicker.vue

```diff
@@ -50,48 +50,50 @@
       :rtl="isRtl"
       :visible="isOpen"
     >
-      <div
-        v-show="isOpen"
-        ref="datepicker"
-        class="vdp-datepicker__calendar"
-        :class="pickerClasses"
-        @mousedown.prevent
-      >
-        <slot name="beforeCalendarHeader" />
-        <Component
-          :is="picker"
-          class="picker-view"
-          :day-cell-content="dayCellContent"
-          :disabled-dates="disabledDates"
-          :first-day-of-week="firstDayOfWeek"
-          :highlighted="highlighted"
-          :is-rtl="isRtl"
-          :is-up-disabled="isUpDisabled"
-          :page-date="pageDate"
-          :selected-date="selectedDate"
-          :show-edge-dates="showEdgeDates"
-          :show-full-month-name="fullMonthName"
-          :show-header="showHeader"
-          :transition-name="transitionName"
-          :translation="translation"
-          :use-utc="useUtc"
-          :view="view || computedInitialView"
-          :year-range="yearPickerRange"
-          @page-change="handlePageChange"
-          @select="handleSelect"
-          @select-disabled="handleSelectDisabled"
-          @set-transition-name="setTransitionName($event)"
-          @set-view="setView"
+      <Transition name="fade">
```

Comment:
Can you make a prop to be able to put your own transition name?

Reply:
I don't think we need a prop for this. It's probably best to rename the transition to something generic; then people can set whatever transition they want in the stylesheet.

At the moment, I'm still working on calculating the height of the popup. It's tedious stuff, but I feel like I made a little progress today. Unfortunately, it only works on `nextTick`, so I'm not sure how nicely this will tie up with the upcoming keyboard navigation I've been working on...